.devcontainer/nvidia/Dockerfile

```diff
@@ -5,11 +5,17 @@ FROM ghcr.io/robotic-decision-making-lab/blue:jazzy-desktop-nvidia
 # add new dependencies during development
 ENV USERNAME=ubuntu
 ENV USER_WORKSPACE=/home/$USERNAME/ws_blue
+ENV USER_GID=1000
+ENV USER_UID=1000
 WORKDIR $USER_WORKSPACE
 
 COPY --chown=$USER_UID:$USER_GID . src/blue
+
+ENV ROS2_LATEST_ARCHIVE_KEYRING="/usr/share/keyrings/ros2-latest-archive-keyring.gpg"
+RUN sudo rm -f ${ROS2_LATEST_ARCHIVE_KEYRING}
+RUN sudo curl -sSL https://raw.githubusercontent.com/ros/rosdistro/master/ros.key -o ${ROS2_LATEST_ARCHIVE_KEYRING}
```

Comment:
This shouldn't be needed. See https://github.com/docker-library/official-images/pull/19162